YogaKit/Tests/YogaKitTests.m

```diff
@@ -384,4 +385,172 @@ - (void)testThatANonLeafNodeCanBecomeALeafNode
     [container.yoga applyLayout];
 }
 
+- (void)testPositionalPropertiesWork
+{
+  UIView *view = [[UIView alloc] initWithFrame:CGRectZero];
+
+  view.yoga.left = 1;
+  XCTAssertEqual(YGNodeStyleGetPosition(view.yoga.node, YGEdgeLeft).value, 1);
```

Comment:
I think we can remove the verification that `YGNodeRef`'s setters are working, that shouldn't be our responsibility.

I do think it's valuable to verify that they are set to `Pixel` values, though.

I do think it's valuable to verify that getter for `view.yoga.*` works, and that it is a `YGUnitPixel` would be more than enough. Any chance we can condense these tests so that we simply test that different